libs/ui/src/lib/components/CtimsAutoCompleteComponent.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { AutoComplete } from 'primereact/autocomplete';
+import axios from 'axios';
+import {getTemplate, getUiOptions, ObjectFieldTemplatePropertyType, ObjectFieldTemplateProps} from "@rjsf/utils";
+import { Tooltip } from 'antd';
+import styles from "./CtimsAutoCompleteComponent.module.css";
+
+const AutocompleteField = ({onChange, ...props }: ObjectFieldTemplateProps) => {
+      const {
+    schema,
+  } = props;
+
+  const [filteredHugoSymbols, setFilteredHugoSymbols] = useState([]);
+  const [selectedHugoSymbol, setSelectedHugoSymbol] = useState([]);
+
+  const searchSymbols = async (gene) => {
+    const query = gene.query;
+    try {
+    const response = await axios.get(`genes?query=${query}`);
+      const symbols = response.data;
+      setFilteredHugoSymbols(symbols);
+    } 
+    catch (error) {
+      console.error('Error:', error);
+    }
+  };
+
+  const arrayContainer: React.CSSProperties = {
+    width: '640px',
+    marginLeft: 'auto',
+  };
+
+  const labelStyle: React.CSSProperties = {
+    height: '36px',
+    fontFamily: "'Inter', sans-serif",
+    fontStyle: 'normal',
+    fontWeight: '400',
+    fontSize: '14px',
+    lineHeight: '7px',
+    color: '#495057',
+  };
```

Comment:
It's also a preference. If these styles are not shared, I usually just leave them at the place where they are used. So you can understand the context around it.

Reply:
I think I'll leave these here from now and await CTIMS team preference